.github/workflows/check-query-files-and-compilation.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
       - name: Check query files (Unix)
         if: matrix.os != 'windows-latest'
-        run: ./nvim.appimage --headless -c "luafile ./scripts/check-queries.lua" -c "q"
+        run: nvim --headless -c "luafile ./scripts/check-queries.lua" -c "q"
```

Comment:
won't this break  linux?

Reply:
If it would it would have broken it already some lines earlier. The file is from master and it already uses `nvim`. I somehow added it via symlinking to path.

Reply:
https://github.com/theHamsta/nvim-treesitter/blob/95e799e13d7e96a10ec10d99c304527fd939f505/scripts/ci-install-ubuntu-latest.sh#L7-L8